src/Altinn.AccessManagement/Program.cs

```diff
@@ -392,22 +393,23 @@ void ConfigurePostgreSql()
             builder.Configuration.GetValue<string>("PostgreSQLSettings:AdminConnectionString"),
             builder.Configuration.GetValue<string>("PostgreSQLSettings:authorizationDbAdminPwd"));
 
-        string workspacePath = Path.Combine(Environment.CurrentDirectory, builder.Configuration.GetValue<string>("PostgreSQLSettings:WorkspacePath"));
-        if (builder.Environment.IsDevelopment())
-        {
-            workspacePath = Path.Combine(Directory.GetParent(Environment.CurrentDirectory).FullName, builder.Configuration.GetValue<string>("PostgreSQLSettings:WorkspacePath"));
-        }
-
         app.UseYuniql(
             new PostgreSqlDataService(traceService),
             new PostgreSqlBulkImportService(traceService),
             traceService,
             new Configuration
             {
-                Workspace = workspacePath,
+                Workspace = Path.Join(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Migration"),
```

Comment:
Nice 👍🏼

Reply:
Are we sure this works? As I remember the reason why the config needs to be different is because execution location is different between local dev and pod execution.

Reply:
I added the msbuild task in file in persistance.csproj

Reply:
now the folder was manually copied into the containe (I also removed that line in docker)

Reply:
@jonkjetiloye